### PR TITLE
fixed a warning

### PR DESCRIPTION
**Pull request by zohabylc**
This is a text

### CODE REVIEW

File: GitHub/GitHub/ViewController.swift [hunk 1]
```diff
@@ -21,9 +21,9 @@ class ViewController: UIViewController {
   }
 
   func mynewawesomefunction() {
-    for i in 1...4 {
-      print("Git is awesome")
-    }
+    
+    print("changed something in code")
```
Comment (Zoha131) [inline GitHub/GitHub/ViewController.swift:25]: What you did here?

Comment (zohabylc) [reply GitHub/GitHub/ViewController.swift:25]: I have fixed the error.

Comment (Zoha131) [inline GitHub/GitHub/ViewController.swift:25]: Change this line.

File: GitHub/GitHub/ViewController.swift [hunk 2]
```diff
@@ -21,9 +21,10 @@ class ViewController: UIViewController {
...
+    print("changed code as per the request")
```
Comment (zohabylc) [inline GitHub/GitHub/ViewController.swift:26]: this is what i wanted

**Comment by zohabylc** [comment]
everything looks great.

**Pull request merged**